src/view/screens/Search/Search.tsx

```diff
@@ -349,7 +491,7 @@ let SearchScreenInner = ({query}: {query?: string}): React.ReactNode => {
         title: _(msg`Top`),
         component: (
           <SearchScreenPostResults
-            query={query}
+            query={queryWithParams}
```

Comment:
Post queries can have params like `lang`, but profiles and feeds don't support these and will return nothing. So those are passed the raw query without params.